app/controllers/old_exams_controller.rb

```diff
@@ -87,11 +87,7 @@ def update
       redirect_to @old_folder
     else
       @old_exam = OldExam.find(params[:id])
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Unscoped call to OldExam#find](https://www.codacy.com/app/lippertsjan/klausurtool-ror/file/2128757496/issues/source?bid=3141297&fileBranchId=3143249#l89)